src/transformers/modeling_utils.py

```diff
@@ -1666,6 +1666,51 @@ def get_memory_footprint(self, return_buffers=True):
             mem = mem + mem_bufs
         return mem
 
+    def to(self, *args):
+        r"""
+        Overrides the `.to` function from `nn.Module` to raise a ValueError if `8-bit` models are tried to be set on a
+        new device or `dtype`
+        """
+        # Checks if the model has been loaded in 8-bit
+        if getattr(self, "is_loaded_in_8bit", False):
+            raise ValueError(
+                "You are assigning your `8-bit` model into a new device or casting it with a new `dtype`. Device and"
+                " `dtype` assignment is not supported for `8-bit` models. Please use the model as it is, since the"
+                " model has already been set to the correct devices and casted to the correct `dtype`."
```

Comment:
Let's be shorter in the first sentence "`to` is not supporter for 8-bit models." then please use the model as it is...

Reply:
Thanks! Fixed in  8931f80